sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/RetryContextOnDiagnosticTest.java

```diff
@@ -739,7 +739,7 @@ public void throttlingExceptionGatewayModeScenario() {
             CosmosException throttlingException = new CosmosException(429, "Throttling Test");
 
             Mockito.when(mockHttpClient.send(Mockito.any(HttpRequest.class), Mockito.any(Duration.class)))
-                .thenReturn(Mono.error(throttlingException), Mono.error(throttlingException),
```

Comment:
why this change?

Reply:
Thanks for catching this, I was testing something and forgot to revert this change in my local, and it got committed.